samples/modules/thrift/hello/client/prj.conf

```diff
@@ -68,3 +68,5 @@ CONFIG_NET_TCP_WORKQ_STACK_SIZE=4096
 CONFIG_NET_MGMT_EVENT_STACK_SIZE=4096
 CONFIG_IDLE_STACK_SIZE=4096
 CONFIG_NET_RX_STACK_SIZE=8192
+
+CONFIG_REQUIRES_MALLOC=y
```

Comment:
Yeah samples should not do this, @cfriedt would have to comment on adding it to the module Kconfig